src/utils/errors.ts

```diff
@@ -15,31 +15,20 @@
 	along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-import * as _Raven from 'raven';
+import * as Sentry from '@sentry/node';
 import { TypedError } from 'typed-error';
 
 import { VERSION } from '.';
 
-export const Raven = _Raven;
-
-Raven.config(process.env.SENTRY_DSN || false, {
-	captureUnhandledRejections: true,
+Sentry.init({
+	dsn: process.env.SENTRY_DSN,
+	debug: process.env.NODE_ENV === 'dev',
```

Comment:
Usually this would be either empty or `development` for dev: http://expressjs.com/en/4x/api.html#app.set

Reply:
Ah, nice catch, ta.